python/tests/test_mode_coeffs.py

```diff
@@ -76,12 +76,12 @@ def run_mode_coeffs(self, mode_num, kpoint_func, nf=1, resolution=15):
         modes_to_check = [1, 2]  # indices of modes for which to compute expansion coefficients
         res = sim.get_eigenmode_coefficients(mflux, modes_to_check, kpoint_func=kpoint_func)
 
-        self.assertTrue(res.kpoints[0].close(mp.Vector3(0.604301, 0, 0)))
-        self.assertTrue(res.kpoints[1].close(mp.Vector3(0.494353, 0, 0), tol=1e-2))
-        self.assertTrue(res.kdom[0].close(mp.Vector3(0.604301, 0, 0)))
-        self.assertTrue(res.kdom[1].close(mp.Vector3(0.494353, 0, 0), tol=1e-2))
-        self.assertAlmostEqual(res.cscale[0],0.50000977,places=5)
-        self.assertAlmostEqual(res.cscale[1],0.50096888,places=5)
+        self.assertTrue(res.kpoints[0].close(mp.Vector3(0.6056758, 0, 0), tol=1e-2))
+        self.assertTrue(res.kpoints[1].close(mp.Vector3(0.4995093, 0, 0), tol=1e-2))
+        self.assertTrue(res.kdom[0].close(mp.Vector3(0.6056758, 0, 0), tol=1e-2))
+        self.assertTrue(res.kdom[1].close(mp.Vector3(0.4995093, 0, 0), tol=1e-2))
+        self.assertAlmostEqual(res.cscale[0],0.5000020,places=2)
+        self.assertAlmostEqual(res.cscale[1],0.5003438,places=2)
```

Comment:
Why did these numbers change recently?

Reply:
These tests were failing likely because of the older precompiled MPB library for Ubuntu 20.04 ([version 1.9.0](https://packages.ubuntu.com/focal/mpb) from June 2019) that (serial) Meep was linking to using the current GitHub Actions CI setup:

https://github.com/NanoComp/meep/blob/a31309269bb212f192213143816cd40bf31bf053/.github/workflows/build-ci.yml#L40

Hopefully, once the latest MPB is built from source in this PR, the original unmodified `test_mode_coeffs.py` tests will pass and these changes can then be reverted.

Reply:
Changes have been reverted since they now seem to be unnecessary.